Gopkg.toml

```diff
@@ -118,7 +118,7 @@ required = [
 ### as we cache dep ensure.
 [[constraint]]
   name = "istio.io/api"
-  branch = "master"
+  revision = "a2d2077b668d34c849d2efeb951da628cb83c466"
```

Comment:
it should really work with master; what happens ?
is there some issue with https://github.com/istio/api/pull/396 ?

Reply:
I was following what is commented 2 lines above:

```
### DO NOT PIN these repos to master. It will result in inconsistent builds in circleci
### as we cache dep ensure.
```